gpsdio_segment/discrepancy.py

```diff
@@ -0,0 +1,122 @@
+import math
+
+inf = float("inf")
+
+
+class DiscrepancyCalculator(object):
+    """Base class that supplies discrepancy calculator"""
+
+    very_slow = 0.35
```

Comment:
It would be nice to have a comment here to know how you determined the `very_slow` value.